config/default.js

```diff
@@ -0,0 +1,189 @@
+const {
+  VERSIONS,
+  RESOURCES
+} = require('@asymmetrik/node-fhir-server-core').constants;
+
+// // Set up whitelist
```

Comment:
Is any of this commented code still relevant? If not, let's just delete it so it doesn't cause confusion